CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 0.20.0 - 2020-11-10
+### Added
+- Create and deploy a .NET 5 web app
+- View properties on a web app
+- Improved extension activation time (by switching to the [azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js))
```

Comment:
Is it worth mentioning the lazy loading here?

Reply:
I think the "what" (activation time is better) is more important than the "how" (azure-sdk-for-js vs. lazy-load) and I think the new azure sdk will have a much bigger impact